SofaKernel/modules/SofaBaseLinearSolver/src/SofaBaseLinearSolver/CGLinearSolver.inl

```diff
@@ -165,7 +157,7 @@ void CGLinearSolver<TMatrix,TVector>::solve(Matrix& M, Vector& x, Vector& b)
             }
 #endif
 
-            /// Compute p = r^2
+            /// Compute ρ = r²
```

Comment:
Is it a good practice to have special characters, even in comments? I don't know